packages/daimo-api/src/contract/foreignCoinIndexer.ts

```diff
@@ -78,6 +79,8 @@ export class ForeignCoinIndexer extends Indexer {
           ])
           .where("chain_id", "=", chainConfig.chainL2.id)
           .where((eb) => eb.between("block_num", "" + from, "" + to))
+          .orderBy("block_num", "asc")
+          .orderBy("sort_idx", "asc")
```

Comment:
does this use the index on the table? or do we need to do
```
.orderBy("block_num", "asc")
.orderBy("tx_idx", "asc")
.orderBy("sort_idx", "asc")

Reply:
order by is nbd (=sorting a small result set), important part is that the WHERE clauses are a prefix of the index